src/stimfit/gui/graph.cpp

```diff
@@ -2152,6 +2163,68 @@ void wxStfGraph::OnDown() {
     Refresh();
 }
 
+void wxStfGraph::ChanUp() {
+    // on Control + Up press, increase the channel number by one and
+    // refresh the window
+    
+    int reference_ch = Doc()->GetSecChIndex();  
+    
+    int channel = Doc()->GetCurChIndex() + 1;
```

Comment:
The `reference_ch` variable is just being used to store the value of the second channel. The scrolling shifts the first channel leaving the reference channel in place.